test/DynamicDependency/data/Microsoft.WindowsAppRuntime.Framework/appxmanifest.xml

```diff
@@ -99,5 +99,17 @@
             <ActivatableClass ActivatableClassId="Microsoft.Windows.Security.AccessControl.SecurityDescriptorHelpers" ThreadingModel="both" />
         </InProcessServer>
     </Extension>
+    <Extension Category="windows.activatableClass.proxyStub">
+        <ProxyStub ClassId="60FC21B2-B396-4D49-94F0-7555869FB93C">
+          <Path>PushNotificationsLongRunningTask.ProxyStub.dll</Path>
```

Comment:
Who generates this proxystub? Shouldn't it be the singleton that generates this?

Reply:
The ProxyStub dll is generated by the dev/PushNotifications/PushNotificationsLongRunningTask.ProxyStub project. We are moving the dll to the framework package so the client/server can pull the ProxyStub from single dependency.